bot0.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func main() {
+	client, err := ethclient.Dial("https://mainnet.infura.io/v3/" + GetAPIKey("INFURA_KEY"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("we have a connection")
+
+	const V2PairAddress string = "0x3356c9a8f40f8e9c1d192a4347a76d18243fabc5"
```

Comment:
This is good. Could you now also print the price if the pair on Uni V3 ?

Reply:
For now I'm printing `SqrtPriceX96` which is the square root of the price devided by 2 to the power of 96